rotkehlchen/balances/historical.py

```diff
@@ -120,6 +121,67 @@ def get_assets_amounts(
 
         return amounts, last_event_id
 
+    def get_historical_netvalue(
+            self,
+            from_ts: Timestamp,
+            to_ts: Timestamp,
```

Comment:
this logic processes only history events. What about trades and asset movements? Those also affect the net worth